pandas/tseries/frequencies.py

```diff
@@ -260,8 +260,7 @@ def infer_freq(index, warn=True):
     if isinstance(index, pd.Index) and not isinstance(index, pd.DatetimeIndex):
         if isinstance(index, (pd.Int64Index, pd.Float64Index)):
             raise TypeError(
-                "cannot infer freq from a non-convertible index "
-                "type {type}".format(type=type(index))
+                "cannot infer freq from a non-convertible index " f"type {type(index)}"
```

Comment:
can you manually concatenate these two literals

Reply:
Sorry, maybe I am misunderstanding you. Do you mean to make them each 1 string instead of 2?

Reply:
yes. just this one. (it's a result of the black reformatting)